src/components/GlobalLearningResourcesPage/GlobalLearningResourcesPage.scss

```diff
@@ -0,0 +1,13 @@
+.lr-c-global-learning-resources-page {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+
+  &__main {
+    display: grid;
+    grid-template-columns: 22% 78%;
```

Comment:
Follow UP, work with UX and get mocks for various screen sizes.

Reply:
Talking to UX atm